src/components/Navigation.tsx

```diff
@@ -116,29 +121,11 @@ export function Navigation({
                                   >
                                     <ItemLink
                                       item={subItem}
+                                      parent={item}
                                       pathname={pathname}
                                       onLinkClick={onLinkClick}
                                     />
                                   </Disclosure.Button>
-                                  {subItem.links && (
-                                    <Disclosure.Panel as="ul">
-                                      {subItem.links.map((tertiary) => (
-                                        <li key={tertiary.title}>
-                                          <Disclosure.Button
-                                            className={`${styles.button} ${styles.tertiary}`}
-                                          >
-                                            <Link
-                                              href={tertiary.href}
-                                              onClick={onLinkClick}
-                                              className={clsx(styles.item)}
-                                            >
-                                              {tertiary.title}
-                                            </Link>
-                                          </Disclosure.Button>
-                                        </li>
-                                      ))}
-                                    </Disclosure.Panel>
-                                  )}
```

Comment:
We don't have tertiary level navigation, so removed it because I haven't tested my changes for tertiary level. We can revisit if we need tertiary navigation or more robust fix.